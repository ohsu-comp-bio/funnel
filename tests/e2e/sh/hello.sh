id=`funnel run --cmd 'echo hello world'`
funnel task wait $id
funnel task get $id
