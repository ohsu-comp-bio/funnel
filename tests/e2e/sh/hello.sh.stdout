{
	"state": "COMPLETE",
	"name": "Funnel run: echo hello world",
	"resources": {

	},
	"executors": [
		{
			"imageName": "alpine",
			"cmd": [
				"echo",
				"hello",
				"world"
			],
			"workdir": "/opt/funnel",
			"stdout": "/opt/funnel/outputs/stdout-0",
			"stderr": "/opt/funnel/outputs/stderr-0"
		}
	],
	"logs": [
		{
			"logs": [
				{
					"stdout": "hello world\n",
				}
			],
		}
	]
}
